tests/talent_options

```diff
@@ -6,112 +6,112 @@ __max_tiers = 7
 __talents_per_tier = 3
 __class_map = {
     'Death_Knight': [
-        (0, -1),        # 56: All
-        (1, -1),        # 57: All
-        (2, -1),        # 58: All
-                        # 60: None
-                        # 75: None
-        (5, -1),        # 90: All
-        (6, -1),        # 100: All
+        (0, -1),        # 15: All
+        (1, -1),        # 25: All
+        (2, -1),        # 30: All
+        (3, -1),        # 35: All
+        (4, -1),        # 40: All
+        (5, -1),        # 45: All
+        (6, -1),        # 50: All
     ],
     'Demon_Hunter': [
-        (0, -1),        # 99: All
-        (1, -1),        # 100: All
-        (2, -1),        # 102: All
-                        # 104: None
-        (4, -1),        # 106: All
-        (5, -1),        # 108: All
-        (6, -1),        # 110: All
+        (0, -1),        # 15: All
+        (1, -1),        # 25: All
+        (2, -1),        # 30: All
+        (3, -1),        # 35: All
+        (4, -1),        # 40: All
+        (5, -1),        # 45: All
+        (6, -1),        # 50: All
     ],
     'Druid': [
         (0, -1),        # 15: All
-                        # 30: None
-        (2, -1),        # 45: All
-                        # 60: None
-        (4, -1),        # 75: All
-        (5, -1),        # 90: All
-        (6, -1),        # 100: All
+        (1, -1),        # 25: All
+        (2, -1),        # 30: All
+        (3, -1),        # 35: All
+        (4, -1),        # 40: All
+        (5, -1),        # 45: All
+        (6, -1),        # 50: All
     ],
     'Hunter': [
         (0, -1),        # 15: All
-        (1, -1),        # 30: All
-                        # 45: None
-        (3, -1),        # 60: All
-                        # 75: None
-        (5, -1),        # 90: All
-        (6, -1),        # 100: All
+        (1, -1),        # 25: All
+        (2, -1),        # 30: All
+        (3, -1),        # 35: All
+        (4, -1),        # 40: All
```

Comment:
I'm not sure about the other classes, but for hunters all of the utility talents are not implemented. So I don't see much point in adding them to the talents test "matrix".

I'm pretty sure this holds true for all class utility rows tbh.

Reply:
Yes, the choice to activate this again was deliberate because talent tiers changed that and should be reduced on a per class-basis. Or are the utilty rows nowadays always in the same rows?

Reply:
They didn't change from bfa to shadowlands at least (and from legion to bfa afair). And I'm pretty sure what we had here was up to date for all classes.

Reply:
Ok, I just removed my changes. The comments do not match, but that does not hurt since we still got the same number of talent rows.